testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/cli/SuiteRun.java

```diff
@@ -124,6 +126,9 @@ public Integer call()
         @Option(names = "--cli-executable", paramLabel = "<jar>", description = "Path to CLI executable " + DEFAULT_VALUE, defaultValue = "${cli.bin}")
         public File cliJar;
 
+        @Option(names = "--impacted-features", paramLabel = "<file>", description = "Only run tests in environments with these features " + DEFAULT_VALUE)
```

Comment:
Would be nice to be able to provide inline input for making use of this locally without having to create a file (or can we use bash process substitution magic for that)?

Reply:
Yes, process substitution works here. The list of impacted features will be very long, that's why we went with passing it as a file, and I doubt it'll be used when executing this manually very often.